### ISSUE TITLE
More complex irods connections

### ISSUE THREAD

**Issue by holtgrewe** [UtrechtUniversity/irods-consistency-check#13]
I think it would be helpful to add `**irods_env` to the `iRODSSession` constructor in `command.py` around line 90.

```
 90     session = iRODSSession(
 91         host=irods_env["irods_host"],
 92         port=irods_env["irods_port"],
 93         user=irods_env["irods_user_name"],
 94         password=password,
 95         zone=irods_env["irods_zone_name"],
 96         **irods_env
 97     )
```

**Comment by stsnel** [comment]
Hello,

We like the idea. Unfortunately I noticed some unexpected side effects when testing the change in our acceptance test environment: ichk fails with an irods.exception.NetworkException after checking a couple thousand data objects when we create the iRODSSession using **irods_env. 

We plan to perform additional tests later to figure out why the change doesn't work as expected, and to see how we can get it to work correctly.

**Comment by stsnel** [comment]
We found the issue that caused the exceptions when using **irods_env to create a new session. If irods_client_server_negotiation is set in irods_environment.json and
passed to iRODSSession, Python-irodsclient will throw an exception when
the script terminates without cleaning up the session properly. This has now been fixed.

A fix for this issue (#15) has been merged into the master branch (Python 3 only) and into the legacy python2 branch .
